NineChronicles.Headless.Executable/Commands/ValidationCommand.cs

```diff
@@ -0,0 +1,45 @@
+using Cocona;
+using Libplanet;
+using Libplanet.Crypto;
+
+namespace NineChronicles.Headless.Executable.Commands
+{
+    public class ValidationCommand : CoconaLiteConsoleAppBase
+    {
+        [Command(Description = "Validate private key")]
+        public int PrivateKey(
+            [Argument(
+                Name = "PRIVATE-KEY",
+                Description = "A hexadecimal representation of private key to validate.")]
+            string privateKeyHex)
+        {
+            try
+            {
+                _ = new PrivateKey(ByteUtil.ParseHex(privateKeyHex));
+                return 0;
+            }
+            catch
+            {
+                return -1;
```

Comment:
IMO, it would be helpful if we leave a related message on stderr for users.

Reply:
I applied it in a5f578e. Could you check it is you intended?